app/src/main/java/at/roteskreuz/stopcorona/constants/Constants.kt

```diff
@@ -61,6 +61,19 @@ object Constants {
         const val INTENSIVE_CONTACT_SCORE = 30.0
     }
 
+    /**
+     * Constants related to the exposure notification framework.
+     */
+    object ExposureNotification {
+
+        /**
+         * 20.18.13.xx
+         */
+        const val MIN_SUPPORTED_GOOGLE_PLAY_APK_VERSION = 201813000
+
+        const val GOOGLE_PLAY_SERVICES_PACKAGE_NAME = "com.google.android.gms"
```

Comment:
you can also use `GoogleApiAvailability. GOOGLE_PLAY_SERVICES_PACKAGE ` instead of hardcoding it
https://developers.google.com/android/reference/com/google/android/gms/common/GoogleApiAvailability#public-static-final-string-google_play_services_package

Reply:
Fixed